guacamole-client/noauth-logged/src/main/java/com/nanocloud/auth/noauthlogged/connection/LoggedConnection.java

```diff
@@ -99,10 +91,7 @@ private void logStartConnection(ActiveConnectionRecord connection, String token)
               .add("user-lastname", lastname)
               .add("connection-id", connection.getConnectionName())
               .add("start-date", connection.getStartDate().toString())
-              .add("end-date", "")
```

Comment:
Does this means `machine-id`, `machine-driver` and `machine-type` in the model History are not used anyone ? Should not those field be removed in a migration script ? This would mean we could no longer link a session history to a machine.